cypress/support/commands.ts

```diff
@@ -0,0 +1,213 @@
+import '@testing-library/cypress/add-commands'
+
+import { baseConfig } from '../../utils'
+import {
+  CHECKOUT_ENDPOINT,
+  DEFAULT_SELLER_ID,
+  getBaseURL,
+} from '../../utils/constants'
+import { fillEmail, fillProfile } from '../../utils/profile-actions'
+
+interface SKU {
+  id: string
+  quantity: number
+  seller?: string
+}
+
+declare global {
+  /* eslint-disable-next-line @typescript-eslint/no-namespace -- Cypress uses
+   * and recommends namespaces to augment the Chainable interface for custom
+   * commands.
+   *
+   * @see https://docs.cypress.io/guides/tooling/typescript-support#Types-for-custom-commands
+   */
+  namespace Cypress {
+    /* eslint-disable-next-line @typescript-eslint/no-unused-vars, @typescript-eslint/no-explicit-any --
+     * All declarations of the Chainable interface must follow the same
+     * generics signature, so we must use `any` as the default value, and
+     * cannot omit the `Subject` generic
+     */
+    interface Chainable<Subject = any> {
```

Comment:
I moved some of our utilities functions into cypress commands, as it makes it easier to create tests without importing a bunch of other functions

I didn't remove the original functions yet because I didn't update the other tests (to make the review a bit easier), but it would be nice to do in the future